compiler/modules/nback/build.bal

```diff
@@ -462,9 +462,9 @@ class Scaffold {
         self.llFunc = llFunc;
         self.diFunc = diFunc;
         self.birBlocks = code.blocks;
-        final Repr[] reprs = from var reg in code.registers select check semTypeRepr(reg.semType);
+        final Repr[] reprs = from var reg in code.registers select check semTypeRepr(reg.semType, err:location(mod.partFiles[defn.partIndex], defn.position));
```

Comment:
Not OK to pass location here. An err:Location is always constructed lazily when we know we have an error.

Reply:
We can either turn the error into a panic. Or if that doesn't work, we can pass a bir:FunctionDefn plus a bir:File[]

Reply:
Changed to panic